### PR TITLE
Splashscreen

### PR DESCRIPTION
**Pull request by Shilupa**

### CODE REVIEW

**Comment by Shilupa** [comment]
Splashscreen and onboarding works well. Can you check it.

File: app/build.gradle [hunk 1]
```diff
@@ -50,6 +50,10 @@ dependencies {
     implementation 'org.jetbrains.kotlin:kotlin-stdlib-jdk7:1.5.31'
     implementation 'androidx.multidex:multidex:2.0.1'
 
+    //page indicator
```
Comment (giaongo) [inline app/build.gradle:53]: why are you doing this

**Review by giaongo — APPROVED** [review]
ok good

**Review by SurajKRB — APPROVED** [review]
Looks cool

**Review by chinguyen202 — APPROVED** [review]

**Pull request merged**